Templates/CRMs/Magento/Magento Products to Salesforce/long-description.md

```diff
@@ -0,0 +1,21 @@
+This template inserts Magento products and their corresponding prices to Salesforce.
+Using this template, you can insert your new Magento products to Salesforce.
+
+![Template](assets/Adobe_Commerce___Magento_Product_to_Salesforce.svg)
+
+# Prerequisites
+
+This template assumes that the following prerequisites are in place:
+
+- The Salesforce user should have access to client ID, secret and security token.
+- Price book for Magento is initialized in Salesforce.
```

Comment:
Maybe we should mention that the name of said price book should be "Magento Price Book" by default 🤔

Reply:
This will be in the next commit!